### PR TITLE
push docker manual with tag

### PR DESCRIPTION
**Pull request by helto4real**
## Proposed change

Adds a manual trigger that pushes current branch to docker using the following naming:
`netdaemon/netdaemon:feature-[tag]`
the tag is set in the manual step.

## Type of change

- [ ] Dependency upgrade
- [ ] Bugfix (non-breaking change which fixes an issue)
- [x] New feature (which adds functionality to an existing integration)
- [ ] Breaking change (fix/feature causing existing functionality to break)
- [ ] Code quality improvements to existing code or addition of tests

## Additional information

- This PR fixes or closes issue: fixes #
- This PR is related to issue:
- Link to documentation pull request:

## Checklist

- [x] The code change is tested and works locally.
- [x] Local tests pass. **Your PR cannot be merged unless tests pass**
- [x] There is no commented out code in this PR.
- [x] I have followed the [development checklist][dev-checklist]
- [x] The code compiles without warnings (code quality chek)
- [ ] Tests have been added to verify that the new code works.

If user exposed functionality or configuration variables are added/changed:

- [ ] Documentation added/updated for [www.home-assistant.io][docs-repository]

[docs-repository]: https://github.com/net-daemon/docs

### CODE REVIEW

**Comment by helto4real** [comment]
Sadly this needs to be merged to dev before test if works :)

File: .github/workflows/push_docker_manual.yml [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+  deploy:
+    name: 📦 Deploy to Docker with tag
+    runs-on: ubuntu-latest
+    environment: CI - release environment
+    steps:
+      - name: 📤 Checkout the repository 
+        uses: actions/checkout@main
+      - name: 📆 Set version number
+        run: |
+          sed -i '/        private const string Version = /c\        private const string Version =  "dev build";' ${{github.workspace}}/src/DaemonRunner/DaemonRunner/Service/RunnerService.cs
```
Comment (BeeHiveJava) [inline .github/workflows/push_docker_manual.yml:21]: Would this work? I'd rather have not care about whitespaces.

```suggestion
          sed -i '/private const string Version = /c\private const string Version = "dev build";' ${{github.workspace}}/src/DaemonRunner/DaemonRunner/Service/RunnerService.cs
```

File: .github/workflows/push_docker_manual.yml [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+          sed -i '/    io.hass.version=/c\    io.hass.version="dev" \\' ${{github.workspace}}/Dockerfile
```
Comment (BeeHiveJava) [inline .github/workflows/push_docker_manual.yml:22]: Same thing here. Not sure if this works.

```suggestion
          sed -i '/io.hass.version=/c\io.hass.version="dev" \\' ${{github.workspace}}/Dockerfile
```

File: .github/workflows/push_docker_manual.yml (hunk 1 above)
Comment (helto4real) [reply .github/workflows/push_docker_manual.yml:21]: Maybe, never tried it without. I will make a separate PR with that suggestion cause I have several actions doing the same thing. But I see now that I forgot to fix the text in the version tag.

**Review by BeeHiveJava — APPROVED** [review]

**Pull request merged**